rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
+function Get2LargestNo(vect)
```

Comment:
Like @balauru mentioned, good code should be self explanatory, comments shouldn't be needed in the majority of cases. A part of what this means to me is that functions and variables should have very clear and descriptive names. (do not use shorthand names)